tripy/docs/post0_developer_guides/how-to-add-new-ops.md

```diff
@@ -192,9 +191,12 @@ from tripy.types import ShapeLike
 # `autodoc_options` parameter.
 @export.public_api(document_under="tensor_operations")
 
-# The `convert_to_tensors` decorator automatically converts compatible
-# arguments, like `TensorLike` or `ShapeLike`s, into tensors.
-@frontend_utils.convert_to_tensors()
+# We can use the `constraints.interface` decorator to specify constraints on
```

Comment:
Might want to rename `constraints` -> `wrappers` or something generic like that. `constraints` doesn't really capture the scope of the decorator.

Reply:
That is true, only part of that file deals with dtype constraints now.